src/features/web-compat.js

```diff
@@ -534,30 +534,38 @@ export default class WebCompat extends ContentFeature {
         // Chrome respects only the last viewport tag
         const viewportTag = viewportTags.length === 0 ? null : viewportTags[viewportTags.length - 1]
         const viewportContent = viewportTag?.getAttribute('content') || ''
-        const viewportContentParts = viewportContent ? viewportContent.split(/,|;/) : []
+        const viewportContentParts = viewportContent ? viewportContent.split(/\s*,\s*|\s*;\s*/) : []
         const parsedViewportContent = viewportContentParts.map((part) => {
             const [key, value] = part.split('=').map(p => p.trim().toLowerCase())
             return [key, value]
         })
 
+        // first, check if there are any forced values
         const { forcedDesktopValue, forcedMobileValue } = this.getFeatureSetting('viewportWidth')
         if (typeof forcedDesktopValue === 'string' && this.desktopModeEnabled) {
             this.forceViewportTag(viewportTag, forcedDesktopValue)
+            return
         } else if (typeof forcedMobileValue === 'string' && !this.desktopModeEnabled) {
             this.forceViewportTag(viewportTag, forcedMobileValue)
-        } else if (!viewportTag || this.desktopModeEnabled) {
+            return
+        }
+
+        // otherwise, check for special cases
+        const forcedValues = {}
+
+        if (this.forcedZoomEnabled) {
+            forcedValues['initial-scale'] = 1
+            forcedValues['user-scalable'] = 'yes'
+            forcedValues['maximum-scale'] = 10
+        }
```

Comment:
Theoretically, it should be enough just to remove `user-scalable=no` if it exists, but I want to mimic the existing implementation which explicitly sets these values